services/ymax-planner/src/engine.ts

```diff
@@ -657,7 +751,11 @@ export const startEngine = async (
       portfolioKeyForDepositAddr,
     });
 
-    await processPendingTxEvents(pendingTxEvents, handlePendingTx, txPowers);
+    await processPendingTxEvents(pendingTxEvents, handlePendingTx, {
+      ...txPowers,
+      error: console.error.bind(console),
+      marshaller,
+    });
```

Comment:
Why is this not using `txPowers` directly? That variable was literally intended for this AFAIK.
```suggestion
    await processPendingTxEvents(pendingTxEvents, handlePendingTx, txPowers);
```

Reply:
`txPowers` does not have `marshaller`.

Reply:
If it wasn't necessary before but is now, then it should be added.